near/nep141-locker/src/lib.rs

```diff
@@ -549,9 +587,94 @@ impl Contract {
 
     fn decode_prover_result(result_idx: u64) -> Result<ProverResult, PromiseError> {
         match env::promise_result(result_idx) {
-            PromiseResult::Successful(data) => Ok(ProverResult::try_from_slice(&data)
-                .unwrap_or_else(|_| env::panic_str("Invalid proof"))),
+            PromiseResult::Successful(data) => {
+                Ok(ProverResult::try_from_slice(&data).sdk_expect("Invalid proof"))
+            }
             PromiseResult::Failed => Err(PromiseError::Failed),
         }
     }
+
+    fn insert_raw_transfer(
+        &mut self,
+        nonce: u128,
+        transfer_message: &TransferMessage,
+        sender_id: &AccountId,
+    ) -> Option<Vec<u8>> {
```

Comment:
We never use the return value. Should we return this?

Reply:
8e64e520b784216e4f62693aa872d294c7b1d3f9